.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish docs via GitHub Pages
```

Comment:
Is it necesarry to introduce an additional CI service: can't we stick to Travis? https://github.com/search?q=mkdocs+gh-deploy+path%3A**%2F.travis.yml&type=code

Reply:
that can probably work as well. On the other hand, this might be a nice oppertunity to try out gh actions.